modules/fedimint-mint-client/src/lib.rs

```diff
@@ -1097,8 +1103,6 @@ impl MintClientModule {
             }
         }
 
-        assert!(!outputs.is_empty());
```

Comment:
I guess we want to support zero-amounts now?

Reply:
What do you mean? This has been removed as due to fees, a small amount, like less then 100 millisats, don create new outputs.

Reply:
That was an interaction between the functions I missed, glad I asked :laughing: